### ISSUE TITLE
Breaks on svelte files with `Uncaught ReferenceError: Buffer is not defined`

### ISSUE THREAD

**Issue by oplik0** [dprint/dprint-plugin-prettier#18]
It seems like all `.svelte` files cause dprint to fail with `Uncaught ReferenceError: Buffer is not defined`. Full error:
```
Error formatting /workspaces/template/src/App.svelte. Message: Uncaught ReferenceError: Buffer is not defined
    at dprint:prettier.js:51811:36
    at String.replace (<anonymous>)
    at snipTagContent (dprint:prettier.js:51807:37)
    at snipScriptAndStyleTagContent (dprint:prettier.js:51791:31)
    at Object.preprocess (dprint:prettier.js:53266:20)
    at Object.m [as parse] (dprint:prettier.js:3311:47)
    at b (dprint:prettier.js:3858:43)
    at D (dprint:prettier.js:3923:194)
    at dprint:prettier.js:11460:136
    at Object.format (dprint:prettier.js:11465:20)
```
This is the result of running `dprint fmt` on https://github.com/sveltejs/template with this minimal configuration:
```json
{
  "prettier": {},
  "includes": [
    "**/*.svelte"
  ],
  "excludes": [
    "**/node_modules"
  ],
  "plugins": [
    "https://plugins.dprint.dev/prettier-0.7.0.json@4e846f43b32981258cef5095b3d732522947592e090ef52333801f9d6e8adb33"
  ]
}
```
But the exact same error seems to appear with any svelte files and with other dprint configurations.

It appears this was a regression in `0.6.0` - older versions of dprint with `0.5.1` (and earlier) work fine, so I'd guess the rewrite to `deno_core` in #14 caused some issue with at least this plugin (I haven't tested astro to see if it's affected too)

**Comment by dsherret** [comment]
Sorry about that! Added an automated test and fixed in 0.7.1

**Comment by oplik0** [comment]
Something still seems broken, but there definitely is progress, as the error message changed :)
Now it's
```
Uncaught ReferenceError: URL is not defined
    at shims/url.js (dprint:prettier.js:26773:27)
    at __require (dprint:prettier.js:29:52)
    at dprint:prettier.js:48680:58
    at dprint:prettier.js:26782:72
    at node_modules/svelte/compiler.js (dprint:prettier.js:26783:9)
    at __require (dprint:prettier.js:29:52)
    at Object.parse (dprint:prettier.js:55309:54)
    at Object.m [as parse] (dprint:prettier.js:5308:84)
    at b (dprint:prettier.js:5855:43)
    at D (dprint:prettier.js:5920:194)
```

I quickly added a cargo test spec in a fork, which is a copy of the one from js tests: https://github.com/oplik0/dprint-plugin-prettier/blob/main/tests/specs/Svelte.txt
And can confirm it fails too: https://github.com/oplik0/dprint-plugin-prettier/runs/6328000246?check_suite_focus=true

Seems something might break when running the code with deno_core instead of full Deno perhaps?

**Comment by dsherret** [comment]
@oplik0 I think so. Thanks! Would you be able to open a PR with your failing test and I will look into fixing it right now?

**Comment by dsherret** [comment]
Ok, hopefully this is fixed now in 0.7.2. The new test at least passes.

**Comment by oplik0** [comment]
Yup, I can confirm that it works for my project too. Well, time to switch it to dprint again, since I didn't expect it to be fixed so quickly and put prettier back in :D
